File: CP/home/arg0.c
#include <stdio.h>
#include <string.h>
int main(int argc,char *argv[])
{
	int i;
	printf("\n %d",argc);
	for(i=0;i<argc;i++)
	{
		printf("\n%s",argv[i]);
	}

	return 0;
}

File: CP/home/arg0.py
import sys



if len(sys.argv)==2:
    print("hello,{}".format(sys.argv[1]))



else:
    print("hello, world")



if len(sys.argv)>2:
    for i in range(len(sys.argv)):
        print(sys.argv[i])

else:
    pass
